pythnet-price-feeds/cosmwasm.md

```diff
@@ -6,9 +6,30 @@ description: Consume Pyth Network prices in applications on Cosmwasm
 
 Cosmwasm contracts can update and fetch the Pyth prices using the Pyth Cosmwasm Contract, deployed on their network. The documented source code can be found [here](https://github.com/pyth-network/pyth-crosschain/tree/main/target_chains/cosmwasm/contracts/pyth).
 
-## Updating Price Feeds
+## Price Feeds
+Pythnet price feeds uses an on-demand price update model, where users are responsible for posting price updates on-chain when needed. The latest price can be fetched from a price service, which is a permissionless centralised service. We recommend protocols to run their own price-service. 
+Pythnet publishes prices for two kinds of feeds: 
+1. Stable price feeds are the most reliable price feeds.
+2. Edge price feeds comes with the latest and greatest features of Pyth price feeds as soon as they are available. 
 
-The mechanism by which price feeds are updated on Cosmwasm is explained [here](./pythnet-price-feeds.md). The [pyth-common-js](https://github.com/pyth-network/pyth-crosschain/tree/main/target_chains/cosmwasm/sdk/js) can be used to fetch the latest price feed data which then can be passed to the `ExecuteMsg` on-chain function.
+Mainnet only supports the stable version of price feeds whereas testnet supports both. 
+
+To learn more about the mechanism by which price feeds are updated on Cosmwasm you can follow this [link](./pythnet-price-feeds.md). The [pyth-common-js](https://github.com/pyth-network/pyth-crosschain/tree/main/target_chains/cosmwasm/sdk/js) can be used to fetch the latest price feed data which then can be passed to the `ExecuteMsg` on-chain function.
```

Comment:
I think this paragraph actually belongs before the `Price Feeds` section (because we usually link to all of the needed SDKs up front)